requirements.txt

```diff
@@ -0,0 +1,5 @@
+uvicorn
+fastapi
+requests
+cryptography
```

Comment:
Specify the versions for the packages you want. It is important because one person can download `uvicorn` now and get `1.1.0` and another one can download it a year from now and will get version `2.0.0` which is incompatible with the current `requests` package.